package.json

```diff
@@ -40,6 +40,7 @@
     "node": ">= 4"
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
Not this discussion again ;) There is only one right answer here: 
**"We should use shrinkwrap to lock versions"**
....

![](http://i2.kym-cdn.com/entries/icons/facebook/000/012/367/evilest.jpg)

More seriously now - Fully agree going to lock down all dependencies, but I needed to mention shrinkwrap here just for fun!